internal/db/schema/migrations/oss/postgres/48/01_vault_credentials.up.sql

```diff
@@ -0,0 +1,256 @@
+begin;
+
+  -- update views
+  drop view credential_vault_store_public;
+  drop view credential_vault_library_public;
+  drop view credential_vault_library_private;
+  drop view credential_vault_store_private;
+
+  create view credential_vault_store_private_active as
```

Comment:
After tracing it down the use of `privateStore` in the code it became clear we have 2 use cases:

1.  Where we want the store with the latest token (current or expired), this is the main use case of the view
2. Where we want all active tokens only used by the vault token renewal and revocation jobs

Instead of trying to get both with a single view and applying `where` clauses throughout the code base, I figured two views would simplify everything. @mgaffney happy to discuss alternatives here.

Reply:
Can you add sql tests for these views?

Reply:
Yes, lets discuss